src/decision/dMCDA.jl

```diff
@@ -45,6 +49,53 @@ function mcda_methods()
     ]
 end
 
+"""
+    mcda_method_names()::Vector{String}
+
+List of MCDA method names.
+"""
+function mcda_method_names()::Vector{String}
+    return [
+        "Cocoso",
+        "Mairca",
+        "Moora",
+        "Piv",
+        "Vikor"
+    ]
+end
+
+"""
+    mcda_method_encoding(mcda_name::String)::Union{Int64, Nothing}
+
+Find the scenario encoding of the given mcda method. Throws `ArgumentError` if method not 
+found.
+"""
+function mcda_method_encoding(mcda_name::String)::Int64
+    method_idx = findfirst(mcda_method_names() .== mcda_name)
+    if isnothing(method_idx)
+        msg = "Given MCDA method $(mcda_name) is not in list of mcda methods. "
```

Comment:
Minor nitpick - space out message onto new line.

```suggestion
        msg = "Given MCDA method $(mcda_name) is not in list of mcda methods. \n"
```